q2cli/builtin/tools.py

```diff
@@ -907,3 +908,249 @@ def cache_status(cache):
     success = "Status of the cache at the path '%s':\n\n%s" % \
         (cache, output)
     click.echo(CONFIG.cfg_style('success', success))
+
+
+@tools.command(name='replay-provenance', cls=ToolCommand)
+@click.option('--in-fp', required=True,
+              help='filepath to a QIIME 2 Archive or directory of Archives')
```

Comment:
```suggestion
              help='filepath to a QIIME 2 Archive (.qza/.qzv) or '
                   'directory of Archives')
```

Useful to define this here, in case the user isn't familiar with the term `Archive` in this context.

Reply:
Not sure if the line break is needed or not for linting.